prescient/simulator/master_options.py

```diff
@@ -638,249 +297,76 @@ def _construct_inner_options_parser():
-                                         action='store',
-                                         type='float',
-                                         dest='relax_ramping_factor',
-                                         default=0.01)
-    guiOverride['--relax-ramping-factor'] = {}
-    guiOverride['--relax-ramping-factor']['bpa'] = False
-    guiOverride['--relax-ramping-factor']['advanced'] = True
-
-    solver_simulation_options.add_option('--error-if-infeasible',
-                                         help='Exit out if an infeasibility is found in SCED / RUC.',
-                                         action='store_true',
-                                         dest='error_if_infeasible',
-                                         default=False)
-    guiOverride['--error-if-infeasible'] = {}
-    guiOverride['--error-if-infeasible']['bpa'] = False
-
-    solver_simulation_options.add_option('--pause-if-infeasible',
-                                         help='Pause after outputting information '
-                                              'if an infeasibility is found in SCED / RUC.',
-                                         action='store_true',
-                                         dest='pause_if_infeasible',
-                                         default=False)
-    guiOverride['--pause-if-infeasible'] = {}
-    guiOverride['--pause-if-infeasible']['bpa'] = False
 
     solver_simulation_options.add_option('--print-sced',
                                          help='Print results from SCED solves.',
                                          action='store_true',
                                          dest='print_sced',
                                          default=False)
-    guiOverride['--print-sced'] = {}
-    guiOverride['--print-sced']['bpa'] = False
-
-    solver_simulation_options.add_option('--ef-mipgap',
-                                         help='Specifies the mipgap for all stochsatic RUC extensive form solves '
-                                              '(including those invoked by PH, if any).',
-                                         action='store',
-                                         dest='ef_mipgap',
-                                         type='float',
-                                         default=0.01)
-    guiOverride['--ef-mipgap'] = {}
-    guiOverride['--ef-mipgap']['bpa'] = False
 
     solver_simulation_options.add_option('--ruc-mipgap',
                                          help="Specifies the mipgap for all deterministic RUC solves.",
                                          action='store',
                                          dest='ruc_mipgap',
                                          type='float',
                                          default=0.01)
-    guiOverride['--ruc-mipgap'] = {}
-    guiOverride['--ruc-mipgap']['bpa'] = False
-
-    solver_simulation_options.add_option('--sced-mipgap',
```

Comment:
Why don't we use sced_mipgap?

Reply:
As defined by Prescient now, SCEDs will always be linear programs, and not mixed-integer linear programs (MIPs).

The only way they would be MIPs in the future would require upstream changes to Egret.